test/BaseFixture.sol

```diff
@@ -114,6 +124,29 @@ contract BaseFixture is Test {
         safe.enableModule(address(delayModule));
         safe.enableModule(address(rolesModule));
 
+        // registering the task in CL automation service
+        deal(LINK, address(safe), LINK_FOR_TASK_TOP_UP);
+        IERC20(LINK).approve(address(CL_REGISTRAR), LINK_FOR_TASK_TOP_UP);
+
+        IKeeperRegistrar.RegistrationParams memory registrationParams = IKeeperRegistrar.RegistrationParams({
+            name: roboModule.name(),
```

Comment:
nice. maybe for production this will need to be appended with a uid (safe addy maybe?)

Reply:
this is a good point!,

will add also that in the test itself. then each upkeep will loke smth like:

`RoboSaverVirtualModule-EURE-0x...`

Reply:
see 9fa49313e2d7e6d82e8517dc803e4f823353fd2a